### PR TITLE
Ifhaam's classes

### PR DESCRIPTION
**Pull request by IfhaamZ**

### CODE REVIEW

File: src/main/java/classes/Order.java [hunk 1]
```diff
@@ -37,17 +31,8 @@ public class Order
     // An instance of Finder class
     private Finder finder = new Finder();
 
-    /*
-     * Constructor Name : Order()
-     * Parameters : title, price, pairing, discount
-     * Description : Alternate Constructor
-     */
-    // public Order(String title, double price, String pairing)
```
Comment (imalisina) [inline src/main/java/classes/Order.java:45]: remove useless comments

**Review by imalisina — CHANGES_REQUESTED** [review]
Also send me an output to show how it is optimized in terminal

File: src/main/java/classes/Order.java (hunk 1 above)
Comment (IfhaamZ) [reply src/main/java/classes/Order.java:45]: Sent Output

Comment (IfhaamZ) [reply src/main/java/classes/Order.java:45]: The screenshot u sent for useless comments was already removed in the given pull request

**Review by imalisina — APPROVED** [review]
Approved

**Pull request merged**